pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarChannelInitializer.java

```diff
@@ -213,17 +213,16 @@ CompletableFuture<Channel> initSocks5IfConfig(Channel ch) {
 
     CompletableFuture<Channel> initializeClientCnx(Channel ch,
                                                    InetSocketAddress logicalAddress,
-                                                   InetSocketAddress resolvedPhysicalAddress) {
+                                                   InetSocketAddress resolvedPhysicalAddress,
+                                                   InetSocketAddress unresolvedPhysicalAddress) {
```

Comment:
We were using this address for `cnx.setRemoteHostName(...);` before. In order to maintain consistency, I think we should consider doing that in this PR too.